install/kubernetes/helm/subcharts/kiali/templates/clusterrole.yaml

```diff
@@ -8,22 +8,40 @@ metadata:
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
 rules:
-- apiGroups: ["", "apps", "autoscaling", "batch"]
+- apiGroups: [""]
   resources:
   - configmaps
-  - cronjobs
-  - deployments
   - endpoints
-  - horizontalpodautoscalers
-  - jobs
   - namespaces
   - nodes
   - pods
   - projects
```

Comment:
I believe this can be deleted. There is no "projects" type in Kubernetes (this is an OpenShift object).

Reply:
You beat me to it :stuck_out_tongue:, done.